src/core/libmaven/mavenparameters.h

```diff
@@ -20,7 +20,8 @@ class Classifier;
 class MavenParameters 
 {
     public:
-        MavenParameters();
+        MavenParameters(string settingsPath="");
```

Comment:
Why an optional argument rather than a separate constructor? Just asking

Reply:
Why have two of them when one is sufficient ? 
If you don't provide the 'settingsPath', which is what happens when mavenParameters is initialized from 'test' module, it will simply load the default settings. Having a different constructor means making the other constructor behave the same way when no 'settingsPath'  is provided, which in this case is handled automatically.

does it make sense ?

Reply:
I prefer the other way. but this is also fine.